.github/workflows/self-scheduled-amd-caller.yml

```diff
@@ -0,0 +1,26 @@
+name: Self-hosted runner (AMD mi210 scheduled CI caller)
+
+on:
+  schedule:
+    - cron: "17 2 * * *"
+  push:
+    branches:
+      - run_amd_scheduled_ci_caller*
+      - rocm-scheduled-ci
+
+jobs:
+  run_amd_ci_mi210:
+    name: AMD mi210
+    if: (cancelled() != true) && ((github.event_name == 'schedule') || ((github.event_name == 'push') && startsWith(github.ref_name, 'run_amd_scheduled_ci_caller')) || ((github.event_name == 'push') && startsWith(github.ref_name, 'rocm-scheduled-ci')))
```

Comment:
The `rocm-scheduled-ci` is temporary just in order to run on this PR: https://github.com/huggingface/transformers/actions/runs/6577295584

Reply:
Done